Figures/Helper.cs

```diff
@@ -0,0 +1,271 @@
+                    }
+                    else
+                    {
+                        figureList.Add(new Trapezoid(width, height, Convert.ToBoolean(Convert.ToInt32(userChoice[isFillParam])), Convert.ToChar(userChoice[figurePointParam])));
+                    }
+
+                    break;
+                default:
+                    Console.WriteLine("Wrong figure type");
+                    break;
+            }
+        }
+
+        static string[] CollectNewFigureData()
+        {
+            const int MAXPARAMSNUM = 4;
+            string[] userFigureParams = new string[MAXPARAMSNUM];
+            const int MINFIGCHOICE = 1, MAXFIGCHOICE = 4;
+            string figurePoint;//, isFilled;
+            Console.WriteLine("Choose the figure you like to add");
+            Console.WriteLine("1. Triangle");
+            Console.WriteLine("2. Rhombus");
+            Console.WriteLine("3. Rectangle");
+            Console.WriteLine("4. Trapezoid");
+            int figureType = Convert.ToInt32(Console.ReadLine());
+            while (figureType < MINFIGCHOICE || figureType > MAXFIGCHOICE)
+            {
+                Console.WriteLine("Wrong input. Try again");
+                figureType = Convert.ToInt32(Console.ReadLine());
+            }
+
+            Console.WriteLine("Would you like to customize the figure?");
+            Console.WriteLine("1. Yes");
+            Console.WriteLine("0. No");
+            int isCustom = Convert.ToInt32(Console.ReadLine());
+            while (isCustom != (int)YN.NO && isCustom != (int)YN.YES)
+            {
+                Console.WriteLine("Wrong input. Try again", isCustom);
+                isCustom = Convert.ToInt32(Console.ReadLine());
+            }
+
+            if (isCustom == (int)YN.YES)
+            {
+                Console.WriteLine("Please enter figure point character");
+                figurePoint = Console.ReadLine();
+                while (figurePoint.Length != 1)
+                {
+                    Console.WriteLine("Wrong input. Try again");
+                    figurePoint = Console.ReadLine();
+                }
+
+                Console.WriteLine("Would you like to fill the figure?");
+                Console.WriteLine("1. Yes \n0. No");
+                
+                int isFilled = Convert.ToInt32(Console.ReadLine());
+                while (isFilled != (int)YN.YES && isFilled != (int)YN.NO)
+                {
+                    Console.WriteLine("Wrong input. Try again");
+                    isFilled = Convert.ToInt32(Console.ReadLine());
+                }
+
+                
+                userFigureParams[2] = isFilled.ToString();
+                userFigureParams[3] = figurePoint;
+            }
+
+            userFigureParams[0] = figureType.ToString();
+            userFigureParams[1] = isCustom.ToString();
+            return userFigureParams;
+        }
+
+        public static void ChangeFigureParameters(List<Figures> figureList)
```

Comment:
The function, by its name, should get the id after asking for it from the user and only change the parameters.